Update 22.02

jayam04 opened the pull request.

jayam04 committed:
Create pkle-fname

jayam04 committed:
Create pkle-lname

jayam04 committed:
Create pkle-email

jayam04 committed:
Create pkle-contact

jayam04 committed:
Update readme.md

jayam04 committed:
Create requirements.txt

jayam04 committed:
Update readme.md

jayam04 committed:
Update readme.md

jayam04 committed:
Update readme.md

jayam04 committed:
Update requirements.txt

jayam04 committed:
Delete PyPrograms/contacts/data directory

jayam04 committed:
Add files via upload

jayam04 committed:
Create tempfile

jayam04 committed:
Add files via upload

jayam04 committed:
Delete pkle-email

jayam04 committed:
Delete pkle-fname

jayam04 committed:
Delete pkle-lname

jayam04 committed:
Delete pkle-number

jayam04 committed:
Delete tempfile

jayam04 committed:
Rename requirements.txt to requirements.md

jayam04 committed:
Rename requirements.md to requirements.txt

jayam04 committed:
Update requirements.txt

jayam04 committed:
Update contacts.py

jayam04 committed:
Add files via upload

jayam04 committed:
Delete pkle-email

jayam04 committed:
Delete pkle-fname

jayam04 committed:
Delete pkle-lname

jayam04 committed:
Delete pkle-number

Commit:
minor changes to output

Commit:
errors.txt

fix to data error, which may occur when program is closed unexpectedly.

Commit:
edits in comments

Commit:
adding feature to disable prompt

Commit:
fixed sorting algo

Commit:
link to @bupboi profile

Commit:
now it has option to delete contact

Commit:
Merge branch 'update-22.02' of https://github.com/JymPatel/Python-FirstEdition into update-22.02

Commit:
now contacts uses key to enable some features

Commit:
updated errors.txt

Commit:
few changes

Commit:
Create functions.cpython-38.pyc

Commit:
Delete functions.cpython-38.pyc

PowerMacFan commented:
what was functions.cpython-38.pyc?

Commit:
spliting main contacts to main and functions

jayam04 commented:
> what was functions.cpython-38.pyc?

i made new file for functions maybe it creates a new file to import it to main

PowerMacFan commented:
ok

On Wed, Feb 23, 2022 at 11:20 PM Jym Patel ***@***.***> wrote:

> what was functions.cpython-38.pyc?
>
> i made new file for functions maybe it creates a new file to import it to
> main
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JymPatel/Python-FirstEdition/pull/45#issuecomment-1049563699>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMITXLY6ST3BVSG4HMPC26DU4XL3VANCNFSM5OZYFRWQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
batteries not included.

PowerMacFan committed:
fixed spelling mistake

PowerMacFan committed:
added and changed some text

Commit:
changes in contacts, areas

remaining triangle area is completed
sorting algorithm moved to functions from main in contacts

Commit:
Merge branch 'update-22.02' of https://github.com/JymPatel/Python-FirstEdition into update-22.02

jayam04 commented:
should we merge it now?
